mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -143,65 +528,101 @@ impl BlockSource for BlockDb {
     }
 }
 
+async fn create_wallet_db(
+    wallet_db_path: PathBuf,
+    consensus_params: ZcoinConsensusParams,
+    check_point_block: Option<CheckPointBlockInfo>,
+    evk: ExtendedFullViewingKey,
+) -> Result<WalletDb<ZcoinConsensusParams>, MmError<ZcoinClientInitError>> {
+    let db =
+        WalletDb::for_path(wallet_db_path, consensus_params).map_to_mm(ZcoinClientInitError::WalletDbInitFailure)?;
+    run_optimization_pragmas(db.sql_conn()).map_to_mm(ZcoinClientInitError::WalletDbInitFailure)?;
+    init_wallet_db(&db).map_to_mm(ZcoinClientInitError::WalletDbInitFailure)?;
+    if db.get_extended_full_viewing_keys()?.is_empty() {
+        init_accounts_table(&db, &[evk])?;
+        if let Some(check_point) = check_point_block {
+            init_blocks_table(
+                &db,
+                BlockHeight::from_u32(check_point.height),
+                BlockHash(check_point.hash.0),
+                check_point.time,
+                &check_point.sapling_tree.0,
+            )?;
+        }
+    }
+    Ok(db)
```

Comment:
As I can see, this function body has been moved from `init_light_client`, where it was wrapped into `async_blocking({})`.
Do we need to use `async_blocking({})` in this function as well?

Reply:
Oh, I missed it during refactoring. Yeap, it should be in `create_wallet_db` function, because `async_blocking` uses `spawn_blocking` from tokio.

Done.